canonical-web-and-design/charmhub.io
# feat: added empty search result view

## Done

Added empty state for the packages page. ([design](https://warthogs.atlassian.net/browse/WD-13856?focusedCommentId=546471))

## How to QA
- Go to https://charmhub-io-2034.demos.haus/
- Enter a search word that returns a result such as "test"
- Verify results are shown
- Apply filters
- Verify proper results are shown
- Enter a search term that doesnt returns any result
- Verify the empty view is shown

## Testing
- [] No testing required (explain why):

## Issue / Card
Fixes [#13856](https://warthogs.atlassian.net/browse/WD-13856)


![Screenshot from 2024-11-19 16-36-26](https://github.com/user-attachments/assets/d6212105-61d0-4d6f-8608-bfbee0d0a8f8)
![Screenshot from 2024-11-19 16-36-59](https://github.com/user-attachments/assets/2cfc755b-47e3-4fd8-87ab-fd0a7a14da39)
